docs/build.md

```diff
@@ -0,0 +1,51 @@
+# Building the Kimai Docker
+
+The same docker file is used to build all the tagged images and is configured by combination of build arguments and 
+targets.
+
+## Targets
+
+The docker file has many intermiadary targets but two functional builds are `prod` and `dev` which correspond to the 
```

Comment:
intermiadary ?

Reply:
https://www.dictionary.com/browse/intermediary

But I'll change it to staging if that's easier to understand.

Reply:
Thanks! Never heard that word before (even though I do know intermediate certificates, if that is even the same word base). The German translation didn't make a lot of sense to me, so I find staging more clear. But that is just me^^